app/src/main/java/eu/kanade/presentation/more/settings/screen/browse/components/ExtensionReposContent.kt

```diff
@@ -55,6 +59,7 @@ fun ExtensionReposContent(
 @Composable
 private fun ExtensionRepoListItem(
     repo: String,
+    onClick: () -> Unit,
```

Comment:
And change it for other in route
```suggestion
    onOpenWebsite: () -> Unit,
```